src/java.base/share/classes/jdk/internal/lang/stable/StableSupplier.java

```diff
@@ -58,7 +58,7 @@ public boolean equals(Object obj) {
 
     @Override
     public String toString() {
-        final Object t = delegate.wrappedContentAcquire();
+        final Object t = delegate.wrappedContentsAcquire();
         return t == this ? "(this StableSupplier)" : StableValueImpl.renderWrapped(t);
```

Comment:
```suggestion
        return delegate.wrappedContentsAcquire() == this 
               ? "(this StableSupplier)" 
               : StableValueImpl.renderWrapped(t);
```
Same as above, simplifying the code by removing local variables that are only used once

Reply:
The `t` variable is used again in the `else` branch of the ternary operator.